src/StoreKeeper/WooCommerce/B2C/Imports/ProductImport.php

```diff
@@ -647,7 +648,14 @@ public static function getAttributeData($product)
                 } else {
                     $attributeData['attribute_names'][$index] = $contentVar->get('label');
                 }
-                $attributeData['attribute_position'][$index] = count($attributeData['attribute_position']);
+
+                if ($contentVar->has('attribute_order')) {
+                    $attributeOrder = $contentVar->get('attribute_order');
+                } else {
+                    $attributeOrder = WooCommerceAttributeMetadata::getMetadata($attribute->id, 'attribute_order', true, 'DESC') ?? 0;
+                }
```

Comment:
Edge cases where users could've synced attributes though tools page, then there would be an `attribute_order` meta data for the attribute but possibly doesn't have it on products' `content_var` yet.

Reply:
I experienced this before you put `attribute_order` and `attribute_option_order` on `content_var`. But if moving forward this will no longer happen, I can remove this. Let me know your thoughts.

Reply:
I think better to read from one place, we won't have problem with timing bugs

Reply:
Removed else clause.